yourplugin.android.ts

```diff
@@ -1,4 +1,4 @@
-import {Common} from './yourplugin.common';
+import {Common} from "./yourplugin.common";
```

Comment:
Minor question and just curious; why move to double-quotes?
I'm somewhat partial to singles; just looks cleaner to me?